instrumentation/spring/spring-webflux-5.0/javaagent/src/test/java/SpringWebfluxTest.java

```diff
@@ -0,0 +1,841 @@
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("/double-greet")
+                        .hasKind(SpanKind.SERVER)
+                        .hasNoParent()
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(NET_TRANSPORT, IP_TCP),
+                            equalTo(NET_SOCK_PEER_ADDR, "127.0.0.1"),
+                            satisfies(NET_SOCK_PEER_PORT, val -> val.isInstanceOf(Long.class)),
+                            equalTo(NET_SOCK_HOST_ADDR, "127.0.0.1"),
+                            equalTo(NET_HOST_NAME, "localhost"),
+                            satisfies(NET_HOST_PORT, val -> val.isInstanceOf(Long.class)),
+                            equalTo(HTTP_TARGET, "/double-greet"),
+                            equalTo(HTTP_METHOD, "GET"),
+                            equalTo(HTTP_STATUS_CODE, 200),
+                            equalTo(HTTP_SCHEME, "http"),
+                            equalTo(HTTP_FLAVOR, "1.1"),
+                            satisfies(HTTP_USER_AGENT, val -> val.isInstanceOf(String.class)),
+                            equalTo(HTTP_ROUTE, "/double-greet"),
+                            satisfies(
+                                HTTP_REQUEST_CONTENT_LENGTH,
+                                val ->
+                                    val.satisfiesAnyOf(
+                                        v -> assertThat(v).isInstanceOf(Long.class),
+                                        v -> assertThat(v).isNull())),
+                            satisfies(
+                                HTTP_RESPONSE_CONTENT_LENGTH,
+                                val ->
+                                    val.satisfiesAnyOf(
+                                        v -> assertThat(v).isInstanceOf(Long.class),
+                                        v -> assertThat(v).isNull()))),
+                span -> {
+                  assertThat(trace.getSpan(1).getName())
+                      .contains(SPRING_APP_CLASS_ANON_NESTED_CLASS_PREFIX, ".handle");
+                  span.hasKind(SpanKind.INTERNAL)
+                      .hasParent(trace.getSpan(0))
+                      .hasAttributesSatisfyingExactly(
+                          satisfies(
+                              stringKey("spring-webflux.handler.type"),
+                              val -> val.contains(INNER_HANDLER_FUNCTION_CLASS_TAG_PREFIX)));
+                }));
+  }
+
+  @ParameterizedTest(name = "{index}: {0}")
+  @MethodSource("provideMultipleDelayingRouteParameters")
+  public void multipleGetsToDelayingRoute(Parameter parameter) {
+    int requestsCount = 50;
+
+    List<AggregatedHttpResponse> responses = new ArrayList<>();
+    IntStream.rangeClosed(0, requestsCount - 1)
+        .forEachOrdered(n -> responses.add(client.get(parameter.urlPath).aggregate().join()));
```

Comment:
Nit: you could use the more idiomatic map->collect way
```suggestion
    List<AggregatedHttpResponse> responses = IntStream.rangeClosed(0, requestsCount - 1)
        .mapToObj(n -> responses.add(client.get(parameter.urlPath).aggregate().join()))
        .collect(Collectors.toList());
```